server/db/models/Product.js

```diff
@@ -1,12 +1,48 @@
 const { STRING } = require('sequelize')
 const db = require('../db')
 
+const Product = conn.define('product', {
+    title: {
+        type: STRING,
+        allowNull: false,
+        validate: {
+            notEmpty: true,a
+        },
+    },
+    brand: {
+        type: STRING,
+        allowNull: false,
+    },
+    category: {
+        type: STRING,
+        validate: {
+            isIn: [['sweet', 'salty', 'healthy']],
+        },
+    },
+    description: {
+        type: Text,
+        allowNull: false,
+    },
+    price: {
+        type: DECIMAL(10, 2),
+        validate: {
+            isDecimal: true,
+        },
+    },
+    inventory: { type: INTEGER },
+    country: { type: STRING },
```

Comment:
country could also use some validators (not null, is not empty)

Reply:
OK!